gds/src/main/scala/edu/gemini/aspen/gds/osgi/Activator.scala

```diff
@@ -31,7 +30,7 @@ class Activator extends BundleActivator {
   private var observationStateEventQ: Queue[IO, ObservationStateEvent] = null
   private var epicsReaderRef: Ref[IO, Option[EpicsReader]]             = null
   private var statusDbRef: Ref[IO, Option[StatusDatabaseService]]      = null
-  private var configDeferred: Deferred[IO, GdsConfiguration]           = null
+  private var configDeferred: Ref[IO, Option[GdsConfiguration]]        = null
```

Comment:
Just curious. Was the `Deferred` not working?

Reply:
It was working fine. I wanted to have a file observer that would restart the process if the config file changes.
This would be useful during commissioning.
However the file watcher requires a newer fs2 version and that seemed more work than what I wanted to do
I'll try to do it later on